src/util/tutorial.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+import type { EdgeAccount } from 'edge-core-js'
+
+const TUTORIAL = 'tutorial.json'
+
+export const getUserLoginCount = async (account: EdgeAccount): Promise<number> => {
+  const { disklet, id } = account
+  try {
+    const userData = JSON.parse(await disklet.getText(TUTORIAL))
+    return userData[id] ? parseInt(userData[id].loginCount) : 0
+  } catch (error) {
+    console.log(error)
+    return 0
+  }
+}
+
+export const setUserLoginCount = async (account: EdgeAccount, loginCount: number): Promise<void> => {
+  const { disklet, id } = account
+  const data = JSON.stringify({ [id]: { loginCount } })
+  await disklet.setText(TUTORIAL, data)
```

Comment:
the tutorial json file needs to be made more flexible for use by other tutorial screens. The way you architected it will make it only work for this one wallet sliding tutorial

Reply:
fixed